src/style.css

```diff
@@ -0,0 +1,62 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+}
+
+.searchContainer {
+  margin-top: 20px;
+}
+
+.searchForm {
+  --text-color: #3d4e61;
+  --font: avenir, helvetica, sans-serif;
+  --gradient: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search {
+  width: 100%;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  font-weight: 900;
+}
+
+.bigSearch {
+  padding: 23px 20px 23px 62px;
+  line-height: 22px;
+  font-size: 16px;
+  background-position: 25px 24px;
+  background-size: 19px;
+}
+
+.smallSearch {
+  padding: 11px 10px 10px 33px;
+  line-height: 19px;
+  font-size: 14px;
+  background-position: 12px 14px;
+  background-size: 11px;
+}
+
+.search::placeholder {
+  font-family: var(--font);
+  font-weight: 300;
+  color: var(--text-color);
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  background: url(images/Search.svg) no-repeat, var(--gradient);
+}
+
+.smallSearch:focus {
+  background: url(images/Search.svg) no-repeat, var(--gradient);
+}
```

Comment:
You haven't set background position here, and size, that's why I see this on focus, fix it
![image](https://user-images.githubusercontent.com/37161315/137161839-af27fcb8-4d2b-4762-ace5-1b1944d5b2b3.png)
![image](https://user-images.githubusercontent.com/37161315/137161890-054a126e-f20d-471f-88e4-5174e167b414.png)